instrumentation/aws-sdk/aws-sdk-1.11/javaagent/aws-sdk-1.11-javaagent.gradle

```diff
@@ -69,11 +71,16 @@ dependencies {
   // needed for kinesis:
   testImplementation group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-cbor', version: versions.jackson
 
+  // needed for SNS
+  testImplementation deps.testcontainers
+  testImplementation "org.testcontainers:localstack:1.15.2"
```

Comment:
Nit: how about adding testcontainers version to the `versions` map and using it here?

Reply:
Done (and used in all other places as well).